src/style.css

```diff
@@ -0,0 +1,128 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --color-blue: #00acdc;
+  --color-dark: #060b35;
+  --color-grey: #616070;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.card__title {
+  margin-top: 40px;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-dark);
+  margin-bottom: 4px;
+}
+
+.card__subtitle {
+  font-style: normal;
+  font-weight: 400;
```

Comment:
default values, so no need to set it (FYI)

Reply:
exactly :)